application/services/charity_service.py

```diff
@@ -0,0 +1,39 @@
+from application.popos.charity import Charity
+import requests
+import os
+from dotenv import load_dotenv
+load_dotenv()
+
+def return_charities(search_term):
+    charities = charity_objects(charity_list(search_term))
+
+    return charities
+
+def charity_objects(response):
+    parsed = response.json()
+    charity_list = []
+    for charity in parsed:
+        id = charity["ein"]
+        name = charity["charityName"]
+        url = charity["charityNavigatorURL"]
+        rating = charity["currentRating"]["rating"]
+        charity_list.append(Charity(id, name, url, rating))
+
+    return charity_list
+
+def charity_list(search_term):
+    payload = {
+                'app_id': os.getenv('CHARITY_APP_ID'),
+                'app_key': os.getenv('CHARITY_APP_KEY'),
+                'rated': 'TRUE',
+                'minRating': 3,
+                'sort': 'RATING:DESC',
+                'search': search_term
+               }
+
+    response = requests.get(
+                            'https://api.data.charitynavigator.org/v2/organizations',
+                            params=payload
+                            )
```

Comment:
Oooo I LOVE this break out of the service and popo files! Shows a clean line of single responsibility.